valve-fsm.txt

```diff
@@ -1,6 +1,12 @@
 //state machine that reads IO and handles valve opened, closed, and transition
 //with failed to open and failed to close states based on timers\
 
+
+#include <stdio.h>
+#include <unistd.h>
+#include <stbool.h>
+
+
```

Comment:
for enums, always give the first one a value, typically 0. They are guaranteed to increment by one, but technically they can start at any value. It's also good practice to have a MAXIMUM_STATE_VAL at the end in case you ever need to do a loop later.

Reply:
done